fastly/domainmanagement/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package domainmanagement contains subpackages which offer various operations to
+// enable, disable, and configure Fastly domains
+package domainmanagement
```

Comment:
I wonder if folks working on this repo prefer to split the word up somehow? Is camel case appropriate? `domainManagement`